src/math/Sphere.js

```diff
@@ -28,10 +28,12 @@ Sphere.prototype = {
 
 	setFromPoints: function () {
 
-		var box = new Box3();
+		var box;
 
 		return function setFromPoints( points, optionalCenter ) {
 
+			box = box || new Box3();
```

Comment:
We prevent circular references all the time using this pattern. Why the inline comment in this case?